packages/store-ui/src/atoms/Overlay/stories/Overlay.stories.tsx

```diff
@@ -17,13 +17,9 @@ const OverlayTemplate: Story<OverlayStoryProps> = ({
     color === 'black' ? { 'data-black': true } : { 'data-green': true }
 
   return (
-    <div data-store-overlay-out-container>
+    <div data-story-overlay-out-container>
```

Comment:
Why is it _story_ here?

Reply:
Yeah, I believe the correct is _store_.

Reply:
I thought story, because this data is just for the story.

Reply:
Yeah, I think `story` here makes sense actually since this is really only being used in this _story_, and is not part of the component's API